modules/sdk/gradle-plugins-workspace/src/main/java/com/liferay/gradle/plugins/workspace/configurator/ClientExtensionProjectConfigurator.java

```diff
@@ -143,6 +146,12 @@ public void apply(Project project) {
 					project, CREATE_CLIENT_EXTENSION_CONFIG_TASK_NAME,
 					CreateClientExtensionConfigTask.class);
 
+		TaskProvider<GenerateLanguageBatchConfigTask>
+			generateLanguageBatchConfigTaskProvider =
+				GradleUtil.addTaskProvider(
+					project, GENERATE_LANGUAGE_BATCH_TASK_NAME,
+					GenerateLanguageBatchConfigTask.class);
```

Comment:
Don't create the provider here - only add it inside the conditional block

Reply:
If you create it here then all CX projects will have this task available, which is not what we want.